web/vue.config.js

```diff
@@ -5,6 +5,19 @@ module.exports = {
     devServer: {
       contentBase: path.join(__dirname, 'dist'),
       writeToDisk: true,
+      // proxy all requests to `/api/` and `/doc` for development.
+      // this fixes the CORS issue for iframe loading and other requests
+      // to the backend
+      proxy: {
+        '^/api': {
+          target: 'http://localhost:5000',
```

Comment:
The default backend port in the README is [8000](https://github.com/docat-org/docat/blob/main/README.md?plain=1#L22), but here it's 5000

Reply:
yes.

Reply:
thats for the docker image, here you starte the local python dev server which runs on port 5000